bpfd/src/bpf.rs

```diff
@@ -400,38 +394,37 @@ impl<'a> BpfManager<'a> {
                     id: id.to_string(),
                     name: p.data.section_name.to_string(),
                     location: p.data.location.to_string(),
-                    program_type: crate::command::ProgramType::Xdp,
-                    attach_type: crate::command::AttachType::NetworkMultiAttach(
-                        crate::command::NetworkMultiAttach {
-                            iface: p.info.if_name.to_string(),
-                            priority: p.info.metadata.priority,
-                            proceed_on: p.info.proceed_on.clone(),
-                            direction: None,
-                            position: p.info.current_position.unwrap_or_default() as i32,
-                        },
-                    ),
+                    program_type: ProgramType::Xdp as i32,
+                    attach_info: crate::command::AttachInfo::Xdp(crate::command::XdpAttachInfo {
```

Comment:
This syntax is funky now... Is there any way we can just make it `ProgramType::Xdp::AttachInfo`

again in 397 we're using the external types defined in `bpfd-api` and then on 398 we're using internal types from `command.rs`

Reply:
> again in 397 we're using the external types defined in bpfd-api

Well actually it's an i32 😉 This used to be an enum in the protobuf spec...
The case for using this here is that while we *could* use the program types from aya (if they're accessible), we actually need TryFrom<i32> and TryFrom<String> for those types (for bpfctl cli and static-programs) which IIRC aren't implemented.